src/main/scala/scalafix/sbt/ScalafixEnable.scala

```diff
@@ -155,21 +155,21 @@ object ScalafixEnable {
                 } else {
                   val latestAvailable =
                     tail.lastOption.getOrElse(earliestAvailable)
-                    Seq(
```

Comment:
This was picked as significant indentation by the compiler (and has been all along by scalafmt as scala3 is used everywhere as dialect), which was causing `latestAvailable`'s type to be recursive :shrug: